### PR TITLE
chore: repairing taxonomies

### PR DESCRIPTION
**Pull request by aleene**
### What
various fixes

### Related issue(s) and discussion
- Fixes #10079

### CODE REVIEW

File: taxonomies/food/categories.txt
Comment (alexgarel) [inline taxonomies/food/categories.txt:4300]: fr:Marne not existing is only a warning, so I don't know if we should remove those.

@stephanegigandet and @teolemon what do you think ?

**Review by alexgarel — COMMENTED** [review]
Incredible work ! :bookmark_tabs: :feather:

I wait for a deliberation on origins before approval !

File: taxonomies/labels.txt
Comment (teolemon) [inline taxonomies/labels.txt:3793]: normally, that's a label for Hauts-de-France @TaciteOFF

File: taxonomies/food/categories.txt
Comment (alexgarel) [reply taxonomies/food/categories.txt:4300]: With @teolemon we think we should not remove this, but maybe, in another PR, add those to origins.txt

Comment (stephanegigandet) [reply taxonomies/food/categories.txt:4300]: Agreed, it would be best to add them to origins.txt

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4117]:
```suggestion
origins:en: en:France, fr:Gascogne
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4131]:
```suggestion
origins:en: en:France, fr:Gascogne
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4141]:
```suggestion
origins:en: en:France, fr:Gascogne
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4247]:
```suggestion
origins:en: en:France, fr:Maine
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4265]:
```suggestion
origins:en: en:France, fr:Jura
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4286]:
```suggestion
origins:en: en:France, fr:Maine
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4300]:
```suggestion
origins:en: en:France, fr:Maine
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4318]:
```suggestion
origins:en: en:France, fr:Côtes-du-Rhône
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4346]:
```suggestion
origins:en: en:France, fr:Coteaux de la Loire
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4353]:
```suggestion
origins:en: en:France, fr:Bugey
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4372]:
```suggestion
origins:en: en:France, fr:Languedoc
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4379]:
```suggestion
origins:en: en:France, fr:Charentes
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4635]:
```suggestion
origins:en: en:France, fr:Antilles françaises
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:4669]:
```suggestion
origins:en: en:France, en:French Guyana
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:5638]:
```suggestion
origins:en: en:France, fr:Saintonge
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:5652]:
```suggestion
origins:en: en:France, fr:Dijon
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:15781]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:16518]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:18269]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:18280]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:19708]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:19777]:
```suggestion
origins:en: en:France, fr:Champagne
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:20157]:
```suggestion
origins:en: en:France, fr:Beaujolais
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:38965]:
```suggestion
origins:en: en:United Kingdom, en:Cornwall
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:44533]:
```suggestion
origins:en: en:France, fr:Pays-Basque
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:45921]:
```suggestion
origins:en: en:United Kingdom, en:Yorkshire
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:54416]:
```suggestion
origins:en: en:France, fr:Cévennes
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:54519]:
```suggestion
# Disabled for safety: origins:en: en:France, fr:Jura
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:54513]:
```suggestion
origins:en: en:France, fr:Gatinais
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:54508]:
```suggestion
origins:en: en:France, fr:Pays de Loire
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:54503]:
```suggestion
origins:en: en:France, fr:Vosges
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:54478]:
```suggestion
# Disabled for safety: origins:en: en:France, fr:Gascogne
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:54423]:
```suggestion
# Disabled for safety: origins:en: en:France, fr:Jura
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:56562]:
```suggestion
origins:en: en:France, fr:Nice
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:56570]:
```suggestion
origins:en: en:France, fr:Nîmes
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:66400]:
```suggestion
origins:en: en:France, fr:Agen
```

File: taxonomies/food/categories.txt
Comment (teolemon) [inline taxonomies/food/categories.txt:105400]:
```suggestion
origins:en: en:France, fr:Nyons
```

File: taxonomies/labels.txt
Comment (teolemon) [inline taxonomies/labels.txt:3793]:
```suggestion
origins:en: en:france, fr:Hauts-de-France
```

**Review by alexgarel — APPROVED** [review]

**Pull request merged**
